src/Core/Grid/Column/Type/Common/PositionColumn.php

```diff
@@ -53,25 +53,25 @@ public function getType()
      */
     protected function configureOptions(OptionsResolver $resolver)
     {
+        parent::configureOptions($resolver);
+
         $resolver
             ->setRequired([
                 'id_field',
                 'position_field',
                 'update_route',
             ])
             ->setDefaults([
-                'sortable' => true,
                 'update_method' => 'GET',
                 'record_route_params' => [],
                 'clickable' => true,
+                'text_align' => 'center',
             ])
             ->setAllowedTypes('id_field', 'string')
             ->setAllowedTypes('position_field', 'string')
             ->setAllowedTypes('update_route', 'string')
-            ->setAllowedTypes('sortable', 'bool')
             ->setAllowedTypes('update_method', 'string')
             ->setAllowedTypes('record_route_params', ['array'])
-            ->setAllowedTypes('clickable', 'bool')
```

Comment:
why remove clickable type?

Reply:
it is a refactoring, see parent::configureOptions($resolver);

https://github.com/tdavidsonas88/PrestaShop/blob/5d38d8cd97a55e6614a48eb04453e691baef1b06/src/Core/Grid/Column/AbstractColumn.php#L112-L125